algotaurus.py

```diff
@@ -521,6 +547,7 @@ def __init__(self, size=15, lines=30):
         self.rt_str.set(_('Run timer: %s msec') % int(self.run_timer))
         self.textPad = tk.Text(self.mainframe, width=15, height=self.lines, wrap='none')
         self.textPad.config(bg='white', fg='black')
+        self.textPad.insert('1.0', self.code)
```

Comment:
Program will be able to store the code from the coder and transfer it a the new mainloop.